recipes/r-sparkr/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set version = '3.1.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-sparkr
+  version: {{ version }}
+
+source:
+  url: https://dist.apache.org/repos/dist/release/spark/spark-3.1.2/SparkR_3.1.2.tar.gz
+  sha256: 1d99b156e2e643fce9a985cc9e64cf9ee5df5f6501292d96f5cc11ec5bc6d0d9
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('SparkR')"           # [not win]
+    - "\"%R%\" -e \"library('SparkR')\""  # [win]
+
+about:
+  home: http://spark.apache.org/
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: Provides an R Front end for 'Apache Spark' <https://spark.apache.org>.
+
+extra:
+  recipe-maintainers:
+    - zero323
```

Can you please also add the conda-forge/r maintainer group. They generally help with updates and migration of R packages (mostly from CRAN). thx.
```suggestion
    - zero323
    - conda-forge/r
```

---

Doe.

---

Sure, done.